src/index.js

```diff
@@ -4,14 +4,14 @@ import { BrowserRouter, Route, Switch } from "react-router-dom";
 
 import registerServiceWorker from "./registerServiceWorker";
 import IndexPage from "./pages/index";
+import Layout from "./layouts/index";
 import Login from "./pages/login";
 import Register from "./pages/register";
-import Layout from "./layouts/index";
 
-import "./index.css";
-
-const Root = () => (
-  <Layout>
+ import "./index.css";
+ 
+ const Root = () => (
+   <Layout>
```

Comment:
This indentation doesn't make sense...

Reply:
It was a copy-paste mistake. I'm gonna run prettier on this page. 
In this case of indentation doesn't fully apply, but next time please elaborate the nonsense by saying the why and how instead of only assert it.